NEWS.md

```diff
@@ -0,0 +1,9 @@
+# EpiEstim 2.1-0
```

Comment:
We've already pushed a release for `2.1-0` so I think this should either read `2.2-0` or `develop`/`master` since these changes will be included in `2.2-0` (which doesn't exist yet). What do you think?

Reply:
Good point! I'll bump the version and triple-check the plots!